src/CMakeLists.txt

```diff
@@ -246,16 +253,20 @@ target_link_libraries(viamcpp
 install(
   TARGETS viamcpp
   EXPORT LibTargets
-  LIBRARY COMPONENT runtime
-  FILE_SET viamcpp_public_includes COMPONENT dev
-  FILE_SET viamcpp_public_pb_includes COMPONENT dev
+  RUNTIME COMPONENT viam-cpp-sdk_runtime
```

Comment:
Changes here are about tagging the new soname symlink and versioned library into the right components, since you don't need the dev symlink in the runtime package.